app/assets/stylesheets/config/_globals.scss

```diff
@@ -116,6 +116,12 @@ a {
 }
 // Components
 
+// Form Components
+
+.form-intput-field {
```

Comment:
Typo here! Should be input, not intput (although that's a good name for a number field!)